android/autobuild/autobuild_apk_2_build.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+
+# autobuild_2_build: actual build process
+
+
+####################
+###  PARAMETERS  ###
+####################
+
+if [ ! -z "${1}" ]; then
```

Comment:
Can't this live somewhere and get sourced?

Reply:
I like the idea.
Only difficulty is a clean way to handle that not all scripts need all parameters.

Reply:
This seems to be done now.